workflow/scripts/get-max-read-length.py

```
@@ -1,5 +1,8 @@
 import json
 import pysam
+import sys
+
+sys.stderr = open(snakemake.log[0], "w")
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)